tensorflow/core/kernels/data/dataset_test_params.h

```diff
@@ -139,102 +195,14 @@ class RangeDatasetParams : public DatasetParams {
 
   RangeDatasetParams(int64 start, int64 stop, int64 step);
 
-  ~RangeDatasetParams(){};
-
   Status MakeInputs(gtl::InlinedVector<TensorValue, 4>* inputs) override;
 
-  Tensor start;
-  Tensor stop;
-  Tensor step;
-};
-
-static std::shared_ptr<BatchDatasetParams> Batch(
```

Comment:
With the updated constructors of these DatasetParams class, the static factories are not needed.